tests/Render.spec.js

```diff
@@ -111,6 +111,9 @@ test('Render', ({assert, done}) => {
 	render(h('img', {width: '100px'}), container)
 	assert(container.firstChild.getAttribute('width') === '100px', 'render element img width')
 
+	render(h('input', {value: 0.4, step: 0.1, type: 'range', min: 0, max: 1}), container)
+	assert(container.firstChild.value === '0.4', 'render element input with un-ordered props')
```

Comment:
TODO: Seems like JSDOM doesn't behave like a real browser when it comes to this behavior so this test always passes irregardless, might have to rethink this test case.

Reply:
consider using [karma](https://github.com/karma-runner/karma) and ChromeHeadless for tests

```
new karma.Server({
    singleRun: true
    frameworks: ['mocha']
    files: ['./tests.js']
    browsers: ['ChromeHeadless']
  }, cb).start()
```